bokeh/charts/gmap.py

```diff
@@ -24,7 +24,7 @@
 #-----------------------------------------------------------------------------
 
 
-class GMap(ChartObject):
+class GMap(Builder):
```

Comment:
Yes.. and no... Gmap is broken for now and I didn't implement it with the new Builders design because I didn't figure out the best way to do it. The problem is that it's not like the others Charts created by builders just creating a list of renderers to be placed on a Chart(Plot). GMap needs GMapPlot (or a GeoJSPlot) instead of a Plot.. so I'm not sure how to deal with it. 

Will try to work more on this PR later today or tomorrow morning. I'm sure I'll think about something :-)